src/api/contact/contact.service.ts

```diff
@@ -1,43 +1,87 @@
+import { ContactDto, EmailVolunteers } from './dto/contact.dto';
 import config from 'src/config/config';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { transporter } from 'src/helpers/nodemailer.helper';
+import * as fs from 'fs';
+import * as ejs from 'ejs';
+import path from 'path';
 
-const { NODEMAILER_PASSWORD, LIBERA_EMAIL_ACCOUNT } = config;
+const { LIBERA_EMAIL_ACCOUNT } = config;
 
 @Injectable()
 export class ContactService {
+  constructor(private prisma: PrismaService) {}
+
   async sendContactForm(contactDto: ContactDto) {
-    // with contactDto.name, contactDto.email, contactDto.message
-    // Create a new nodemailer transporter with the appropriate SMTP credentials
-    const transporter = nodemailer.createTransport({
-      service: 'gmail',
-      auth: {
-        user: LIBERA_EMAIL_ACCOUNT,
-        pass: NODEMAILER_PASSWORD,
-      },
+    const template = fs.readFileSync(
+      path.join(__dirname, '../../../../src/helpers/email-templates/contactEmailTemplate.ejs'),
+      'utf-8',
+    );
+
+    const renderedTemplate = ejs.render(template, {
+      name: contactDto.name,
+      email: contactDto.email,
+      message: contactDto.message,
+      phone: contactDto.phone,
+      company: contactDto.company,
     });
 
     // Set up the email message using the "Get in Touch" form data
     const message = {
       from: LIBERA_EMAIL_ACCOUNT,
       to: LIBERA_EMAIL_ACCOUNT,
       subject: 'Libera Test',
-      text: `
-        Name: ${contactDto.name}
-        Email: ${contactDto.email}
-        Message: ${contactDto.message}
-      `,
+      html: renderedTemplate,
     };
 
     try {
       // Send the email using the nodemailer transporter
       await transporter.sendMail(message);
-      console.log('Email sent successfully');
       return true;
     } catch (error) {
       console.error(error);
       return false;
     }
   }
+
+  async emailVolunteers(emailVolunteersInput: EmailVolunteers) {
+    const template = fs.readFileSync(
+      path.join(__dirname, '../../../../src/helpers/email-templates/volunteerEmailTemplate.ejs'),
+      'utf-8',
+    );
+
+    const volunteers = await this.prisma.projectUser.findMany({
+      where: {
+        projectId: emailVolunteersInput.projectId,
+      },
+      include: {
+        user: true,
+        project: true,
+      },
+    });
+
+    for (const volunteer of volunteers) {
+      const renderedTemplate = ejs.render(template, {
+        volunteerName: volunteer.user.name,
+        body: emailVolunteersInput.body,
+        projectName: volunteer.project.name,
+      });
+
+      const message = {
+        from: LIBERA_EMAIL_ACCOUNT,
+        to: volunteer.user.email,
+        subject: `LIBERA - ${volunteer.project.name} - ${emailVolunteersInput.subject}`,
+        html: renderedTemplate,
+      };
+
+      try {
+        await transporter.sendMail(message);
+      } catch (error) {
+        console.error(error);
```

Comment:
What if we log this? Instead of printing the console log? Is it un mucho texto? To discuss, because, maybe it is too much.